### PR TITLE
Update strapi detection

### PR DESCRIPTION
**Pull request by ysm-dev**
- https://www.patriziagarganti.com/
image

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -5471,7 +5471,8 @@
     ],
     "description": "Strapi is an open-source headless CMS used for building fast and easily manageable APIs written in JavaScript.",
     "headers": {
-      "X-Powered-By": "^Strapi"
+      "X-Powered-By": "^Strapi",
+      "x-powered-by": "^Strapi"
```
Comment (nurbek91) [inline src/technologies/s.json:5475]: hey mate, i think headers work case insensitive

Comment (raystinc) [reply src/technologies/s.json:5475]: > hey mate, i think headers work case insensitive

but Strapi isn't detected on this site: https://www.patriziagarganti.com/

Do you know what is the reason?

Comment (ysm-dev) [reply src/technologies/s.json:5475]: > hey mate, i think headers work case insensitive

but Strapi isn't detected on this site: https://www.patriziagarganti.com/

Do you know what is the reason?

Comment (nurbek91) [reply src/technologies/s.json:5475]: yes mate, header parameter check only first file which at the top, other files we can check with only xhr parameter

Comment (ysm-dev) [reply src/technologies/s.json:5475]: oh I see, didn't know that. Thanks!

**Pull request closed without merging**